util/helpers.py

```diff
@@ -3,9 +3,42 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import argparse
+import zlib
 from pathlib import Path
 
 
+def file_crc(file_path: Path) -> str:
+    """Calculates CRC32 of the provided file
+
+    Args:
+        file_path: The path of the file to calculate the CRC32.
+
+    Returns:
+        String containing the CRC32 of the file.
+    """
+    crc_str = 0
```

Comment:
The naming of the variable is ambiguous. `zlib.crc32()` [returns an unsigned 32-bit integer](https://docs.python.org/3/library/zlib.html#zlib.crc32).

Reply:
Fixed.